airbyte-cdk/python/setup.py

```diff
@@ -58,12 +58,10 @@
         "requests_cache",
         "Deprecated~=1.2",
         "Jinja2~=3.1.2",
-        "cachetools",
     ],
     python_requires=">=3.9",
     extras_require={
         "dev": [
-            "freezegun",
```

Comment:
These were removed accidentally - they were part of the commits that shouldn't have been reverted. Need to be added back in like the changelog. This was a merge conflict resolution issue. Adding them back in!